test/unit/util/webhooks.test.ts

```diff
@@ -0,0 +1,87 @@
+import { calculateProcessingTimeInSeconds } from "../../../src/util/webhooks";
+
+let dateNowSpy;
+
+const currentTime = 1632726221873;
+
+beforeAll(() => {
+	// Lock Time
+	dateNowSpy = jest.spyOn(Date, "now").mockImplementation(() => currentTime);
+});
+
+afterAll(() => {
+	// Unlock Time
+	dateNowSpy.mockRestore();
+});
+
+describe("Webhooks suite", () => {
+	const mockInfoLog = jest.fn();
+	const mockContextLogger = {
+		info: mockInfoLog,
+	};
+
```

Comment:
We might want to spy/mock statsd here and verify that the hystogram methods were called with the expected value